File: Fluid.MvcSampleVB/Startup.vb
Imports Microsoft.Extensions.DependencyInjection
Imports FluidMvcViewEngine
Imports Microsoft.AspNetCore.Builder
Imports Microsoft.AspNetCore.Hosting
Imports Microsoft.Extensions.Logging

Public Class Startup
    Shared Sub New()
        TemplateContext.GlobalMemberAccessStrategy.Register(Of Person)()
    End Sub

    Public Sub ConfigureServices(services As IServiceCollection)
        services.AddMvc().AddFluid()
    End Sub

    Public Sub Configure(app As IApplicationBuilder, env As IHostingEnvironment, loggerFactory As ILoggerFactory)
        If env.IsDevelopment() Then
            app.UseDeveloperExceptionPage()
        End If

        app.UseMvc(Sub(routes)
                       routes.MapRoute(name:="default",
                                        template:="{controller=Home}/{action=Index}/{id?}")
                   End Sub)
    End Sub
End Class

File: Fluid.MvcSample/Startup.cs
using Fluid.MvcViewEngine;
using Microsoft.AspNetCore.Builder;
using Microsoft.AspNetCore.Hosting;
using Microsoft.Extensions.DependencyInjection;
using Microsoft.Extensions.Hosting;

namespace Fluid.MvcSample
{
    public class Startup
    {
        // This method gets called by the runtime. Use this method to add services to the container.
        // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
        public void ConfigureServices(IServiceCollection services)
        {
            services.Configure<FluidMvcViewOptions>(options =>
            {
                options.Parser = new CustomFluidViewParser(new FluidParserOptions());
                options.TemplateOptions.MemberAccessStrategy = UnsafeMemberAccessStrategy.Instance;
            });

            services.AddMvc().AddFluid();
        }

        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
        {
            if (env.IsDevelopment())
            {
                app.UseDeveloperExceptionPage();
            }

            app.UseRouting();

            app.UseEndpoints(routes =>
            {
                routes.MapControllerRoute(
                    name: "default",
                    pattern: "{controller=Home}/{action=Index}/{id?}");
            });
        }
    }
}
